packages/SwingSet/src/kernel/kernelSyscall.js

```diff
@@ -64,14 +76,79 @@ export function makeKernelSyscallHandler(tools) {
     kernelKeeper.incStat('syscalls');
     kernelKeeper.incStat('syscallVatstoreSet');
     kvStore.set(actualKey, value);
+    clearVatStoreIteration();
     return OKNULL;
   }
 
+  function vatstoreGetAfter(vatID, priorKey, lowerBound, upperBound) {
```

Comment:
I think that's the wrong way to think about it.  `priorKey` uses the empty string to mean "the key value that is prior to any other key value".  `upperBound` uses `undefined` following the JavaScript convention for a default parameter (it's not assigned in the parameter list declaration because the actual computation of the default value is a bit more complicated than fits nicely into a parameter declaration).  I suppose it might be possible to declare it using something like `function vatstoreGetAfter(priorKey, lowerBound, upperBound = computeDefaultUpperBound(lowerBound))` except that the actual computation of the default wants to be a layer or two farther down the call stack.